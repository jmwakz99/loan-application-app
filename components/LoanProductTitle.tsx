import { styles } from "@/styles/loanProductTitle";
import { FC } from "react";
import { Text } from "react-native";

type Props = {
  title: string;
};

const LoanProductTitle: FC<Props> = ({ title }) => {
  return <Text style={styles.titleText}>{title}</Text>;
};

export default LoanProductTitle;
