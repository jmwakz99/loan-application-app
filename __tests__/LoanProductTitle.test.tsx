import { render } from "@testing-library/react-native";

import LoanProductTitle from "@/components/LoanProductTitle";

describe("LoanProductTitle", () => {
  it("it renders corectly", () => {
    const title = "Test Title";

    const { getByText } = render(<LoanProductTitle title={title} />);

    const loanProductTitle = getByText("Test Title");

    expect(loanProductTitle).toBeTruthy();
  });

  it("snapshot", () => {
    const title = "Test Title";

    const loanProductTitle = render(<LoanProductTitle title={title} />);

    expect(loanProductTitle).toMatchSnapshot();
  });
});
