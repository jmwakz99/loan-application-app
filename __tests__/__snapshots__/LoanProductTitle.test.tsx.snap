// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoanProductTitle snapshot 1`] = `
<Text
  style={
    {
      "color": "#000000",
      "fontFamily": "",
      "fontSize": 40,
      "fontWeight": "bold",
    }
  }
>
  Test Title
</Text>
`;
